nodejs/week1/week1homework/src/bakckend/routes/large-meals.js

```diff
@@ -0,0 +1,17 @@
+function getLargeMeals(){
+    
+    mealArr=require("../data/meals.json");
+  reviewArr=require("../data/reviews.json");
```

Comment:
This is not the correct way to initialize a variable. You need to use let/const/var. In this case, since these variables are only bound once, you should use `const`.